src/components/BarSelectionBrands/index.js

```diff
@@ -0,0 +1 @@
+export { default as BarSelectionBrands } from './BarSelectionBrands';
```

Comment:
New line.
```suggestion
export { default as BarSelectionBrands } from './BarSelectionBrands';

```

Always put a new line at the end of the file